### PR TITLE
Upgrade to Polymer 3

### PR DESCRIPTION
**Pull request by rslawik**
Run polymer modulizer and manually google-apis dependency and some formatting issues.

I renamed `tests` to `test` so the `polymer test` command can be used.

### CODE REVIEW

**Comment by rslawik** [comment]
@e111077 could you have a look?

File: demo/demo-elements.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+$_documentContainer.innerHTML = `<dom-module id="demo-element">
+  <template id="page-template">
+    <style>
+      div {
+        margin-bottom: 1em;
+      }
+    </style>
+    <h1><code>&lt;google-youtube&gt;</code> Demo</h1>
+    <h3>Full API Demo</h3>
+    <google-youtube id="googleYouTube" playsupported="{{playSupported}}" video-id="mN7IAaRdi_k" state="{{state}}" currenttime="{{currentTime}}" currenttimeformatted="{{currentTimeFormatted}}" duration="{{duration}}" durationformatted="{{durationFormatted}}" fractionloaded="{{fractionLoaded}}" volume="{{volume}}" playbackrate="{{playbackRate}}" playbackquality="{{playbackQuality}}" on-google-youtube-state-change="handleStateChange" on-google-youtube-error="handleYouTubeError">
```
Comment (e111077) [inline demo/demo-elements.js]: This is a demo, can you please beautify the attributes on this file?

File: demo/index.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
     <title>&lt;google-youtube&gt; Demo</title>
-    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="demo-elements.html">
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
```
Comment (e111077) [inline demo/index.html]: for webcomponents.org purposes, please point this to the newest unpkg url of this file

File: test/google-youtube-basic.html [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
     <title>google-youtube Basic Tests</title>
-    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <script src="../../web-component-tester/browser.js"></script>
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../../../wct-browser-legacy/browser.js"></script>
 
-    <link rel="import" href="../../polymer/polymer.html">
-    <link rel="import" href="../google-youtube.html">
+    <script type="module" src="../../../@polymer/polymer/polymer-legacy.js"></script>
```
Comment (e111077) [inline test/google-youtube-basic.html]: delete this line

File: test/google-youtube-basic.html [hunk 4]
```diff
@@ -4,11 +4,11 @@
...
+    <script type="module" src="../google-youtube.js"></script>
```
Comment (e111077) [inline test/google-youtube-basic.html]: delete this line

File: test/google-youtube-custom-thumbnail.html [hunk 5]
```diff
@@ -4,12 +4,12 @@
...
-    <script src="../../web-component-tester/browser.js"></script>
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../../../wct-browser-legacy/browser.js"></script>
 
-    <link rel="import" href="../../polymer/polymer.html">
-    <link rel="import" href="../../test-fixture/test-fixture.html">
-    <link rel="import" href="../google-youtube.html">
+    <script type="module" src="../../../@polymer/polymer/polymer-legacy.js"></script>
+    <script type="module" src="../../../@polymer/test-fixture/test-fixture.js"></script>
+    <script type="module" src="../google-youtube.js"></script>
```
Comment (e111077) [inline test/google-youtube-custom-thumbnail.html]: delete this block of script type modules

File: demo/index.html (hunk 2 above)
Comment (e111077) [reply demo/index.html]: https://unpkg.com/@webcomponents/webcomponentsjs@2.2.1/webcomponents-bundle.js

File: test/google-youtube-custom-thumbnail.html (hunk 5 above)
Comment (rslawik) [reply test/google-youtube-custom-thumbnail.html]: Done

File: test/google-youtube-basic.html (hunk 4 above)
Comment (rslawik) [reply test/google-youtube-basic.html]: Done

File: test/google-youtube-basic.html (hunk 3 above)
Comment (rslawik) [reply test/google-youtube-basic.html]: Done

File: demo/index.html (hunk 2 above)
Comment (rslawik) [reply demo/index.html]: Done.
Thanks for providing the url.

**Comment by rslawik** [comment]
Yes, the demo works.

`http://127.0.0.1:8081/components/@google-web-components/google-youtube/demo/` renders correctly.

Tested with:
```
polymer serve --module-resolution=node --npm
```

File: demo/demo-elements.js (hunk 1 above)
Comment (rslawik) [reply demo/demo-elements.js]: Done. Wrapped lines longer than 100 characters.

**Comment by rslawik** [comment]
Done, deleted.

**Comment by e111077** [comment]
published

**Comment by rslawik** [comment]
Thanks @e111077!
It works! :)
https://jsbin.com/topedatoyu/edit?html,output

**Comment by e111077** [comment]
Dead. I am dead.

**Pull request merged**